README.md

```diff
@@ -29,6 +29,29 @@ Statistical analysis and plotting is performed using the 'statsplotting' script.
 This script uses the importfromjson function to import the JSON format file exported by pygwytracing and calculates various statistical parameters for all grain quantities, e.g. length, width and saves these out as a new JSON file using the savestats function. 
 Both KDE plots and histograms are generated for any of the grain quantities using the matplotlib42 and seaborn43 libraries within the functions plotkde, plotcolumns and plothist. 
 
+## Run using Docker
+
+Docker conatiners provide a way to run software in a controlled environment, without having to install lots of packages on your computer. To experiment with TopoStats using Docker you will first need to install [Docker](https://docs.docker.com/get-docker/). Then, using the command line, "pull" the container image to your computer:
+
+```
+docker pull afmspm/topostats:wip
+```
+
+Then run the container, mapping your current folder (the root of this repository) to a folder in the container (e.g.):
+
+```
+docker run -it -v C:\Users\bobturner\Documents\TopoStats:/home/TopoStats afmspm/topostats:wip
+```
+
+Your next commands will be executed within the container:
+
+```
+cd home/TopoStats
+xvfb-run python pygwytracing.py
```

Comment:
https://stackoverflow.com/a/25280523 might be useful if you want to access the GUI from the Docker host.

Reply:
Alternatively, if you're running a 'X' display system server on the Docker host you could add `-e DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix:rw` to the Docker command-line so programs in the container can display UIs using that X display server.  It's less portable/versatile than the above (VNC-based) approach though.

Reply:
Sure - however, I feel like this could be the subject of a future pull request.